compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/FlushCoroutineDispatcher.skiko.kt

```diff
@@ -87,4 +92,12 @@ internal class FlushCoroutineDispatcher(
             isPerformingRun = false
         }
     }
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    override fun scheduleResumeAfterDelay(timeMillis: Long, continuation: CancellableContinuation<Unit>) {
+        scope.launch {
```

Comment:
```suggestion
        val block = Runnable { continuation.resume(Unit, null) }
        synchronized(tasksLock) {
            delayedTasks.add(block)
        }
        scope.launch {
            kotlinx.coroutines.delay(timeMillis)
            performRun {
                val isTaskAlive = synchronized(tasksLock) {
                    delayedTasks.remove(block)
                }
                if (isTaskAlive) {
                    block.run()
                }
            }
        }
```

We should add `delayedTasks` property, similar to `tasks`, and use it in `hasTasks` method (but not in `flush`). Otherwise, a test can stop work before it ends all scheduled work.

Reply:
done